src/Website/Resources/thankyou.php

```diff
@@ -43,28 +43,21 @@
                 <div class="nav-linkwrapper">
                     <a href="../index.php">Home</a>
                 </div>
-
-                <div class="nav-linkwrapper">
-                    <a href="order.php">Order Form</a>
-                </div>
                 <div class="nav-linkwrapper">
                     <a href="logout.php">Logout</a>
                 </div>
-                <!-- 
-                <div class="nav-linkwrapper">
-                    <a href="login.html">Login</a>
-                </div>
-                -->
-
             </div>
         </div>
     </div>
     <div class="content-wrapper">
        <div class="order-form-wrapper">
             <h1>Thank you for your order!</h1>
-            <h1>Your order of </h1><?php echo htmlspecialchars($_SESSION["quantity"]); ?><h1> </h1><?php echo htmlspecialchars($_SESSION["itemid"]); ?><h1>is being processed</h1>
+            <h1>Your order of <?php echo htmlspecialchars($_SESSION["quantity"]); ?> <?php echo htmlspecialchars($_SESSION["itemid"]); ?> is being processed</h1>
+       </div>
+       <h1>\nTo place another order click <a href="order.php">here</a> </h1>
+       <div class="portfolio-item-wrapper">
+           <div class="portfolio-img-background" style="background-image:url(images/sharkdab.jpg)"></div>
```

Comment:
Added some style with that Shark Dab image.

Reply:
hell yeah!